web/packages/core/src/ruffle-player.ts

```diff
@@ -734,25 +735,29 @@ export class RufflePlayer extends HTMLElement {
         }
 
         if (this.loadedConfig?.defaultFonts?.sans) {
-            this.instance!.set_default_font(
+            builder.setDefaultFont(
                 "sans",
                 this.loadedConfig?.defaultFonts.sans,
             );
         }
         if (this.loadedConfig?.defaultFonts?.serif) {
-            this.instance!.set_default_font(
+            builder!.setDefaultFont(
                 "serif",
                 this.loadedConfig?.defaultFonts.serif,
             );
         }
         if (this.loadedConfig?.defaultFonts?.typewriter) {
-            this.instance!.set_default_font(
+            builder!.setDefaultFont(
                 "typewriter",
                 this.loadedConfig?.defaultFonts.typewriter,
             );
         }
```

Comment:
Is this exclamation syntax here necessary? (Doesn't seem to be used a few lines up)

Reply:
Fixed; I made it iterate the keys and do it automatically, too. It turns out we forgot to add japanese fonts to this list, which this also fixes.